_ont_xgs/ont-huawei-hn8010ts.md

```diff
@@ -11,20 +11,146 @@ parent: Huawei
 | ----------- | -------- |
 | Vendor      | Huawei   |
 | Model       | HN8010Ts |
-| Chipset     |          |
+| Chipset     | HiSilicon SD5117  |
 | Manufacter  |          |
-| Flash       |          |
-| RAM         |          |
-| System      |          |
+| Flash       | 128MB    |
+| RAM         | 32MB     |
+| System      | Dopra Linux  |
 | 10GBaseT    | Yes      |
-| Optics      |          |
-| IP address  |          |
-| Web Gui     |          |
+| Optics      | SC/APC   |
+| IP address  | 192.168.100.1  |
+| Web Gui     |  ✅ user: `telecomadmin`, password: `admintelecom` |
 | SSH         |          |
-| Telnet      |          |
+| Telnet      | After Enabling (via XML file) user: `root`, password: `adminHW` |
 | Serial      |          |
 | Form Factor | ONT      |
 
 ## Software Version
 
-- V5R02C0C10S165
\ No newline at end of file
+- V5R02C0C10S165
+
+## List of partitions
+
+0x000000000000-0x000000100000 : "bootcode" (1MiB)  
+0x000000100000-0x000008000000 : "ubilayer_v5" (127MiB)  
+
+# Use
+
+## Enabling telnet
+
+We need access to a specific shell and in order to get it we have to enable telnet. On the unit, the telnet server accepts connections but doesn't read any input, nor does it send anything.
+
+To enable telnet we’ll use one of the web interface’s features: the configuration backup and restore page. The configuration can be downloaded as an XML, which contains a lot more options than the web interface and a couple of them can be modified to enable telnet access: under the `X_HW_CLITelnetAccess` node, set the `Access` attribute to `1` and in `AclServices`, `TELNETLanEnable` should also be set to `1`. I had to set both but your mileage may vary. 
+
+It is possible that this won't work/has been fixed in more recent versions.
+
+Now that this is done, the modified configuration file can be uploaded via the web interface, the ONT will restart and telnet should be accessible on port 23.
+
+
+# General Settings and Useful Commands
+
+{% include alert.html content="All results need to be converted from HEX to ASCII. You can use our [Converter](/ascii-hex), taking care to add 0x in front and remove the spaces every 8 hex digits." alert="Warning" icon="svg-info" color="blue" %}
+
+## Getting the OLT vendor
```

Comment:
could you make a dedicated page to import since these instructions seem to be the same as the huawei hg8010h?

Reply:
what do you mean? the procedure is exactly the same

Reply:
i've create ont-huawei-useful-command.md file and put into _ont directory, please check if the reference URL is corret for XGS ONT and also for HG8010